packages/core/primitives/signals/src/graph.ts

```diff
@@ -179,6 +179,11 @@ export interface ReactiveNode {
    * Called when a signal is read within this consumer.
    */
   consumerOnSignalRead(node: unknown): void;
+
+  /**
+   * A debug name for the reactive node. Used in Angular DevTools to identify the node.
+   */
+  debugName?: string;
```

Comment:
```suggestion
  debugName: string|undefined;
```
and add a value of `undefined` to `REACTIVE_NODE`.